src/Installer/VendorInstaller.php

```diff
@@ -83,7 +80,7 @@ public function install()
 
         $this->execute->mustRunAll($processes);
 
-        $this->getOutput()->step('Install npm dependencies');
+        $this->getOutput()->step('Install nmp dependencies');
```

Comment:
nmp?

Reply:
Heh, I was trying to modify that string to mention Node.js, but decided to keep as is at the end.